node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminStateUpdater.java

```diff
@@ -64,6 +75,18 @@ public NodeAdminStateUpdater(
         this.orchestrator = orchestrator;
         this.dockerHostHostName = dockerHostHostName;
         this.lastTick = clock.instant();
+
+        storageMaintainer.ifPresent(maintainer -> specVerifierScheduler.scheduleWithFixedDelay(() -> {
+            if (currentState != RESUMED) return;
+
+            try {
+                String hardwareDivergence = maintainer.getHardwardDivergence();
+                NodeAttributes nodeAttributes = new NodeAttributes().withHardwareDivergence(hardwareDivergence);
+                nodeRepository.updateNodeAttributes(dockerHostHostName, nodeAttributes);
+            } catch (RuntimeException e) {
+                log.log(Level.WARNING, "Failed to report hardware divergence", e);
```

Comment:
If getting hardware divergence (from spec verifier) fails, it already increments maintenance failure count (https://github.com/vespa-engine/vespa/blob/273a3b5e1d754ad6ca321488a761b0dcb7df48d9/node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/StorageMaintainer.java#L325), but it re-throws the exception. The only unaccounted for exception is if updating node-repo fails.